README.org

```diff
@@ -78,7 +78,11 @@ on the system, you can set the env var =SYSTEM_RE2=1=. If you do that
 and the library can not be found, it will fall back to a local copy
 (=c_src/re2=). Also, if you want to override the RE2 version that is
 fetched and built, when not using system RE2, you can do so by setting
-=RE2_REV= to a git rev.
+=RE2_REV= to a git rev.   By default, the re2 upstream code will be
+fetched from =https://github.com/google/re2= or in some cases
+=https://code.googlesource.com/re2=, however, if you need to fetch the
```

Comment:
Should we use explain when this happens? If the re2 nif project was cloned from github, then we clone RE2 from github.com/google, assuming that host is accessible. If not, we use the googlesource.com remote, especially for availability reasons.

Reply:
Seems like a lot of details, that are probably not necessary?   It might be better as a comment in the script.  In other words if someone really needs this they will likely look into how the source is pulled anyway?

Reply:
Agreed, how about making it even simpler as follows?

By default, RE2 will be fetched from Google's git repository, however, if you need to fetch the[...]

The part _Google's git repository_ could be improved?